src/PageContent/FoodLogs/AddDeliveryModal.js

```diff
@@ -0,0 +1,223 @@
+        }
+
+        let pickedUpInfo = {
+            signature: event.target.signedByDonating.value,
+            timestamp: timestamp,
+            temperature: event.target.temperature.value
+        }
+
+        let deliverers = [];
+        deliverers.push(deliverer1);
+        deliverers.push(deliverer2);
+
+        let description = {
+            foodItems: this.state.currentItems
+        }
+
+        let daContact = {
+            name: event.target.donatingContactName.value,
+            phone: event.target.donatingContactPhone.value
+        }
+
+        let delivery = {
+            daContact: daContact,
+            donatingAgency: event.target.donatingName.value, 
+            delivererGroup: event.target.delivererGroup.value,
+            receivingAgency: event.target.receivingAgency.value,
+            type: 'recurring',
+            status: DeliveryStatus.COMPLETED,
+            timezone: this.props.account.timezone,
+            raContact: raContact,
+            deliverers: deliverers,
+            description: description,
+            endTimestamp: timestamp,
+            startTimestamp: timestamp,
+            notes: event.target.notes.value,
+            deliveredInfo: deliveredInfo,
+            pickedUpInfo: pickedUpInfo,
+            manuallyAdded: true
+        }
+
+        return delivery;
+    }
+
+    render() {
+        return(
+            <div className="dialog-wrapper">
+                {/* close dialog if anywhere outside of dialog is clicked */}
+                <div className="dialog-overlay" onClick={this.props.closeModal}></div> 
+                <dialog id="manual-dialog" className={"event-dialog dialog-radius" + (!this.state.continue ? " small-dialog" : " big-dialog")} open>
+                    { !this.state.continue ? 
+                    <div>
+                        <div className="add-delivery-warning"><span className="red-text">Warning:</span> If your past delivery was already recorded by volunteers through Meal Matchup when they were delivering it you don't have to add it in again. Only add your delivery if you don't already see it in your history. </div>
+                        <div className="warning-button-group">
+                            <button className="warning-button" onClick={() => this.setState({continue: true})}>Continue</button>
+                            <button className="warning-button" onClick={this.props.closeModal}>Exit</button>
+                        </div>
+                    </div> :
+                    <div>
+                        <div className="d-flex modal-header">
+                            <h1>Add Delivery Details</h1>
+                            <button onClick={this.props.closeModal} className="close-button"><img src={close} alt="close modal" className="close-button-img"></img></button>
+                        </div>
+                        <p className="past-delivery-margin-p">Leave input blank if unsure.</p>
```

Comment:
Maybe add this because when it says leave input blank if unsure I thought that meant I could sumbit with nothing but you need a few fields.
Date*
Time*

* Indicate the field must be filled in